addons/event_sale/models/sale_order.py

```diff
@@ -22,6 +24,18 @@ def action_confirm(self):
                     .for_xml_id('event_sale', 'action_sale_order_event_registration')
         return res
 
+    def action_view_attendee_list(self):
+        return {
+            "name": _("Attendees"),
+            "type": "ir.actions.act_window",
+            "res_model": "event.registration",
+            "view_mode": "tree",
+            "domain": [('sale_order_id', '=', self.id)]
+        }
+
+    def _compute_attendee_count(self):
+        self.attendee_count = len(self.env['event.registration'].search([('sale_order_line_id', 'in', self.order_line.ids)]))
```

Comment:
Beware computed fields should work in batch. You should probably use a read_group (see existing code, there are a lot of them in odoo :) ) .